purchases/src/main/kotlin/com/revenuecat/purchases/Offerings.kt

```diff
@@ -95,5 +96,6 @@ internal fun Offering.withPresentedContext(placementId: String?, targeting: Offe
         metadata = this.metadata,
         availablePackages = updatedAvailablePackages,
         paywall = this.paywall,
+        paywallComponents = this.paywallComponents,
```

Comment:
This was causing some issues with the `current` offering not having paywall components.